functions/EffectSelectPage.ts

```diff
@@ -5,6 +5,10 @@ export interface Props {
   pageIds: number[];
 }
 
+export interface Return {
+  selectedPageIds: number[];
+}
+
```

Comment:
Does this interface is used?

Reply:
No but I would like to use it in the future! I think Candeia will solve it with the new engine